packages/server/src/core/TeamSpeak/plugins/second-instance/afk_kick.ts

```diff
@@ -1,51 +1,48 @@
-const Ts3 = require('../../../TeamSpeak');
-const log = require('../../../../utils/log');
-const { convertToMiliseconds } = require('../../../../utils/time');
-const Cache = require('../../../Cache');
+import { Ts3 } from '@core/TeamSpeak';
+import { convertToMiliseconds } from '@utils/time';
+import log from '@utils/log';
+import Cache from '@core/Cache';
 
-var loaded = false;
+let loaded: ReturnType<typeof setInterval>;
 
-module.exports.main = async () => {
-	const { data } = this.info.config;
+export const main = async function (): Promise<void> {
+	const { data } = info.config;
 	const clients = await Ts3.clientList({ client_type: 0 });
 	clients.forEach(async (client) => {
-		if (client.isAfk(data.minTime) && client.cid !== data.dest) {
+		if (client.isAfk(data.interval) && client.cid !== data.dest) {
```

Comment:
This may be changed with minTime, but also may conflict with ts Plugin Config type

Reply:
We will leave it like it does. This will be another breaking change.